libs/message-coder/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@votingworks/message-coder",
+  "version": "1.0.0",
+  "description": "Encode and decode messages for binary communication.",
+  "keywords": [],
+  "license": "AGPL-3.0",
+  "author": "VotingWorks Eng <eng@voting.works>",
+  "main": "build/index.js",
+  "scripts": {
+    "build": "tsc --build tsconfig.build.json",
+    "clean": "rm -rf build tsconfig.tsbuildinfo tsconfig.build.tsbuildinfo",
+    "lint": "pnpm type-check && eslint .",
+    "lint:fix": "pnpm type-check && eslint . --fix",
+    "pre-commit": "lint-staged",
+    "test": "is-ci test:ci test:watch",
+    "test:ci": "jest --ci --coverage --reporters=default --reporters=jest-junit --maxWorkers=7",
+    "test:coverage": "jest --coverage",
+    "test:watch": "TZ=UTC jest --watch",
+    "type-check": "tsc --build"
+  },
+  "dependencies": {
+    "@votingworks/types": "workspace:*",
+    "@votingworks/utils": "workspace:*",
```

Comment:
I think you're only using `@votingworks/utils` for `typedAs`. I don't think @jonahkagan currently has that moved in #2940 but I think it would make sense to move it in #2940. CC @jonahkagan.

Reply:
Sure, I can do that. I erred on the side of not including anything I was unsure about in libs/basics on first pass, but this seems reasonable. Feel free to request other things you think should be moved as well

Reply:
I think it would make sense for it to be in `@votingworks/basics`, yeah.

Reply:
Done in #2940

Reply:
Updated to use `@votingworks/basics`.